plugins/advtable.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Advanced Tables plugin
+title_nav: Advanced Tables
+description: Add sorting functionality to tables.
+keywords: advtable premium
+---
+
+The `advtable` plugin is a premium plugin that extends the core [`table` plugin]({{site.baseurl}}/plugins/table/) by adding sort options for rows and columns. Tables can be sorted by row or column values using:
+* The Sort options in the Table menu.
```

Comment:
There's also the table toolbar button that exposes sort.

Reply:
added